zyte_common_items/processors.py

```diff
@@ -124,7 +124,7 @@ def brand_processor(value: Any, page: Any) -> Any:
         value = value.strip()
         return Brand(name=value) if value else None
 
-    if isinstance(value, (Selector, SelectorList, HtmlElement)):
+    if isinstance(value, (Selector, HtmlElement)):
```

Comment:
bug, `value` can't be `SelectorList` after `value = _handle_selectorlist(value)`